xhttp/retriable/retriable_test.go

```diff
@@ -1,4 +1,4 @@
-package retriable_test
```

Comment:
@dissoupov i modified this to be consistent with all other packages in dolly. Also need it to be able to access some unexported variables for unit testing.